### PR TITLE
Combine api key actions

### PR DESCRIPTION
**Pull request by JellyWang-3**
Took out redundant code and combined 2 API key actions(reloadApiKey and fetchApiKey) to 1.

### CODE REVIEW

File: web/src/components/DandiAppBar.vue [hunk 1]
```diff
@@ -171,7 +171,7 @@ export default {
       return !(this.name && this.description);
     },
     ...mapGetters(['loggedIn', 'user']),
-    ...mapState(['apiKey', 'girderRest']),
+    ...mapState(['apiKey', 'girderRest', 'status']),
```
Comment (AlmightyYakob) [inline web/src/components/DandiAppBar.vue:174]: Is having `status` here a mistake? It's not in the store and isn't used here either.

Comment (JellyWang-3) [reply web/src/components/DandiAppBar.vue:174]: Yeah, I forgot to take it out. It's supposed to be in #137 branch that I'm working on rn.

Comment (AlmightyYakob) [reply web/src/components/DandiAppBar.vue:174]: Cool, once that's removed I think this looks good.

Comment (JellyWang-3) [reply web/src/components/DandiAppBar.vue]: Once it passes all the tests, can I merge and close the issue or should I wait?

Comment (AlmightyYakob) [reply web/src/components/DandiAppBar.vue]: Yep, for the record #138.

**Comment by JellyWang-3** [comment]
Fixes #138

**Pull request merged**